### ISSUE TITLE
feature: user advice if a hash throws 'token length exception'

### ISSUE THREAD

**Issue by roycewilliams** [hashcat/hashcat#3262]
The hashcat 'token length exception' message is one of the top sources of user confusion and questions.

Proposed user feedback improvement:

* When a hash throws 'token length exception' during hash processing, increment a counter
* After hash processing is complete, if this counter is non-zero, show an "advice" level warning about token length exceptions

Potential language:
```
* Token length exception: [hashcount] / [total hashcount] hashes
  This error happens if the wrong hash type is specified, if the hashes are
  malformed, or if input is otherwise not as expected (for example, if the
  --username option is used but no username is present)
```
(The text has been formatted to wrap under 80 columns)

**Comment by roycewilliams** [comment]
If accumulating a counter isn't worth the trouble, @Chick3nman suggested just flipping to true a global flag that defaults to false. We could then just check if it's true and make the message "One or more hashes produced a 'token length exception' error." for the first line.

**Comment by jsteube** [comment]
OK, I'll create it in such a way that if we decide to add info messages also for other error types (for instance "Line-length exception") we can easily do so.

**Comment by roycewilliams** [comment]
Fixed in 5977edb1d2cc1d1ad4c045b8b5f056078446728b - looks great, thanks!

**Comment by davidbolvansky** [comment]
I did not check it but please ensure that this info will not be printed when machine_readable == true.
